src/v/cluster/topics_frontend.cc

```diff
@@ -959,7 +959,11 @@ topics_frontend::partitions_with_lost_majority(
                 it.check();
             }
         }
-        // todo (next commit): add validation.
+        auto validation_err
+          = _topics.local().validate_force_reconfigurable_partitions(result);
+        if (validation_err) {
+            co_return errc::concurrent_modification_error;
```

Comment:
maybe we can return a `validation_err` here ?

Reply:
this was intentional because if a user makes a get_majority_lost_partitions() they don't expect an error unless something concurrently modified the state while processing the request (hence concurrent_modification_error).. this validation_error is only possible in that situation.